bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerMetadata.java

```diff
@@ -197,7 +197,7 @@ boolean hasPassword() {
         return Arrays.copyOf(password, password.length);
     }
 
-    BookKeeper.DigestType getDigestType() {
+    public BookKeeper.DigestType getDigestType() {
```

Comment:
why is this made public?

Reply:
I was for tests outside the same package.
I will use reflection. it is better